code/modules/projectiles/boxes_magazines/internal/shotgun.dm

```diff
@@ -44,7 +44,9 @@
 /obj/item/ammo_box/magazine/internal/shot/improvised
 	name = "improvised shotgun internal magazine"
 	ammo_type = /obj/item/ammo_casing/shotgun/improvised
+	ammo_type = /obj/item/ammo_casing/shotgun_20bore // New improvised shotgun fires 20 gauge instead of 12 gauge.
```

Comment:
Why does this have two of the same var? This not the way to do this. Either it needs to be using a list or you need to make improvised 20 bore shells.

Have a look at this pr as an example of how you might implement that: https://github.com/Citadel-Station-13/Citadel-Station-13/pull/10641

Reply:
Oops, a mistake from copying the code to atomise the PR into multiple smaller ones, thanks for pointing it out.